Subject: createvm: prioritize riscv64
From: Heinrich Schuchardt heinrich.schuchardt@canonical.com Tue Jan 21 22:52:40 2025 +0100
Date: Mon Mar 3 10:13:14 2025 -0500:
Git: 6c6c39be2fc26713b9756e668af7ab8106e7acae

As all major distros support the riscv64 architecture, add it to the
list of prioritized architectures. This will move it up in the
architecture drop-down menu and thereby easier to find when creating
a new VM.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>

diff --git a/virtManager/createvm.py b/virtManager/createvm.py
index e37921603..1a8c6b4a1 100644
--- a/virtManager/createvm.py
+++ b/virtManager/createvm.py
@@ -792,7 +792,7 @@ class vmmCreateVM(vmmGObjectUI):
         archs.sort()
 
         prios = ["x86_64", "i686", "aarch64", "armv7l", "ppc64", "ppc64le",
-            "s390x"]
+            "riscv64", "s390x"]
         if self.conn.caps.host.cpu.arch not in prios:
             prios = []  # pragma: no cover
         for p in prios[:]:
